### ISSUE TITLE
Dividers in settings

### ISSUE THREAD

**Issue by yiyin** [fruitflybrain/neu3d#38]
It might be useful to put some dividers (instead of folders which needs to unfold them to see them) to group, for example, opacity settings for highlight mode, opacity settings for pinned mode, size for radius, size for soma radius, size for synapse. This can make the settings for opacity and size a bit more understandable.

**Comment by TK-21st** [comment]
What do you mean by divider? I'm not sure what you have in mind.

**Comment by yiyin** [comment]
Like this:
<img width="392" alt="image" src="https://user-images.githubusercontent.com/656268/166943730-21356beb-0216-4de2-bce7-c108ed2f5059.png">

**Comment by TK-21st** [comment]
Ah I see. This may require some custom code, I dont think dat.gui supports this kind of visual organization natively.

**Comment by yiyin** [comment]
Just created them as folders and they are by default are open when the `size` folder is opened. see 1097c84.

**Comment by TK-21st** [comment]
> Just created them as folders and they are by default are open when the `size` folder is opened. see [1097c84](https://github.com/fruitflybrain/neu3d/commit/1097c84988b94047eac05f19db7335d4e59a5a30).

Great! I just checked it out and it looks good. I was thinking if you want to add those labels as part of the panel and not as folders that'd be a bit more work.

On a side note, I find the label "Change" a bit confusing. Maybe change it to "Rerender on Change"? It may not fit on the text box but perhaps something similar. 
![image](https://user-images.githubusercontent.com/26802334/167015020-f96088df-be9d-489e-9215-1aade7f577a8.png)

**Comment by TK-21st** [comment]
Actually I see that the full name is "Change existing?" I think the issue is with tooltip not being available for dat.gui.

I've added a tooltip for this entry, you can do it basically by 
```javascript
f0.add(this.settings, 'neuron3dApp').name("Change existing?").title("Change existing neurons' rendering mode?");
```
where the `.title` adds the tooltip.

I've added a few more to clarify some confusing items but feel free to add more if deemed necessary.

**Comment by TK-21st** [comment]
closed by 1097c84988b94047eac05f19db7335d4e59a5a30
